src/components/MainView/main-view.jsx

```diff
@@ -37,22 +44,37 @@ class MainView extends React.Component {
     }, () => {
       // TODO: Manually enter a patient in context if Patient resource fails to return
     }).then(() => {
-      // TODO: Set UI loader property to false
-
-      // Below is just a test to see if service exchange works. Normally, this should go into the
-      // patient view component
-      const defaultPatientService = 'https://fhir-org-cds-services.appspot.com/cds-services/patient-hello-world';
-      callServices(defaultPatientService);
+      this.props.setLoader(false);
     });
   }
 
   render() {
+    const hookView = this.props.hook === 'patient-view' ? <PatientView /> : <div>Med Prescribe View</div>;
```

Comment:
Does the else part need to be a DIV ? Can we do just a string response ? This could help reduce extra unnecessary DIV's.